css/style.css

```diff
@@ -15,5 +14,195 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+h1 {
+  margin: auto;
+  margin-top: auto;
+  font-size: 40px;
+}
+h2 {
+  margin: auto;
+  margin-top: auto;
```

Comment:
the `margin` property above sets that margin on all 4 sides of the box, so no need to set again with `margin-top`